forge/test/conftest.py

```diff
@@ -448,3 +448,23 @@ def pytest_runtest_logreport(report):
         for key, default_value in environ_before_test.items():
             if os.environ.get(key, "") != default_value:
                 os.environ[key] = default_value
+
+
+def pytest_collection_modifyitems(config, items):
+
+    marker = config.getoption("-m")  # Get the marker from the -m option
+
+    if marker and marker == "model_analysis":  # If a marker is specified
```

Comment:
Please check if this is correct, marker can also be an expression and "not model_analysis" or "push and nightly" are all valid -m options, how will 
```if marker and marker == "model_analysis```
work in that case?